controllers/grid/users/author/AuthorGridHandler.inc.php

```diff
@@ -227,11 +223,29 @@ function getRequestArgs() {
 	}
 
 	/**
-	 * Determines if there should be an 'add user' action on this grid.
-	 * Overridden by child grids.
+	 * Determines if there should be add/edit actions on this grid.
+	 * @param $user User
 	 * @return boolean
 	 */
-	function canAdminister() {
+	function canAdminister($user) {
```

Comment:
I think that centralizing here this logic you also fixed a bug that were always adding the ordering feature to ROLE_ID_SUB_EDITOR even if they were not assigned to the submission... Nice.